pkg/cmd/diagnostics/diagnostics_list.go

```diff
@@ -0,0 +1,55 @@
+package diagnostics
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dustin/go-humanize"
+
+	diagnostics "github.com/dcos/dcos-core-cli/pkg/diagnostics/v2"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+)
+
+func newDiagnosticsListCommand(ctx api.Context) *cobra.Command {
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List available bundles",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := pluginutil.HTTPClient("")
+			client := diagnostics.NewClient(c)
```

Comment:
```suggestion
			client := v2.NewClient(c)
```
I wonder why the checks didn't fail for this PR...

Reply:
Why should it fail? `diagnostics` referred to the `v2` package in the import, which is ok I think.

Reply:
Ah, didn't see that import. All good. I was never really here.